src/am-client.ts

```diff
@@ -0,0 +1,248 @@
+import Axios from 'axios';
+import { OutgoingHttpHeaders } from 'http';
+import * as shortid from 'shortid';
+import * as url from 'url';
+
+export interface AmServerInfo {
```

Comment:
Is it worth having these interfaces in a seperate file?

Reply:
yes